smoke/tests/image_test.go

```diff
@@ -140,6 +140,52 @@ func (i *ImageTestSuite) TestConvertAndCopyImage(t *testing.T, ctx tool.Context,
 	tool.RunWithoutOutput(t, checkCmd)
 }
 
+func (i *ImageTestSuite) TestGenerateChunkdict() test.Generator {
+    return func() (name string, testCase test.Case) {
+		imagename1 := "redis:7.0.1"
+		imagename2 := "redis:7.0.2"
+		imagename3 := "redis:7.0.3"
+		image1 := i.prepareImage(i.T, imagename1)
+		image2 := i.prepareImage(i.T, imagename2)
+		image3 := i.prepareImage(i.T, imagename3)
+        ctx := tool.DefaultContext(i.T)
+
+        // Prepare work directory
+        ctx.PrepareWorkDir(i.T)
+        defer ctx.Destroy(i.T)
+
+        logLevel := "--log-level warn"
+        nydusifyPath := ctx.Binary.Nydusify
+
+		target1 := fmt.Sprintf("%s-nydus-%s", image1, uuid.NewString())
+		target2 := fmt.Sprintf("%s-nydus-%s", image2, uuid.NewString())
+		target3 := fmt.Sprintf("%s-nydus-%s", image3, uuid.NewString())
+        convertCmd1 := fmt.Sprintf(
+			"%s %s convert --source %s --target %s --nydus-image %s --work-dir %s",
+			ctx.Binary.Nydusify, logLevel, image1, target1, ctx.Binary.Builder, ctx.Env.TempDir,
+		)
+		tool.RunWithoutOutput(i.T, convertCmd1)
+		convertCmd2 := fmt.Sprintf(
+			"%s %s convert --source %s --target %s --nydus-image %s --work-dir %s",
+			ctx.Binary.Nydusify, logLevel, image1, target2, ctx.Binary.Builder, ctx.Env.TempDir,
+		)
+		tool.RunWithoutOutput(i.T, convertCmd2)
+		convertCmd3 := fmt.Sprintf(
+			"%s %s convert --source %s --target %s --nydus-image %s --work-dir %s",
+			ctx.Binary.Nydusify, logLevel, image1, target3, ctx.Binary.Builder, ctx.Env.TempDir,
+		)
+		tool.RunWithoutOutput(i.T, convertCmd3)
+        target := fmt.Sprintf("%s,%s,%s", target1, target2, target3)
+       
+		generateCmd := fmt.Sprintf(
+			"%s %s chunkdict generate --sources %s --nydus-image %s --work-dir %s",
```

Comment:
nydus-image chunkdict generate --database ?
Besides, a verify by read db file is required for correctness.

Reply:
Only smoke test has been implemented for `nydusify chunkdict generate` command, database is only used in nydus-image, so there isn't